examples/two_functions_with_dispatch.md

```diff
@@ -0,0 +1,192 @@
+
+## Full container bytecode
+
+```
+ef0001 030006 01003b 010017 01001d 00 000001010101 60043560003560e01c63c766526781145d001c63c6c2ea1781145d00065050600080fd50fb000260005260206000f350fb000160005260206000f3 600181115d0004506001fc60018103fb000181029050fc 600281115d0004506001fc60028103fb000260018203fb0002019050fc
+```
+
+Without spaces:
+```
+ef000103000601003b01001701001d0000000101010160043560003560e01c63c766526781145d001c63c6c2ea1781145d00065050600080fd50fb000260005260206000f350fb000160005260206000f3600181115d0004506001fc60018103fb000181029050fc600281115d0004506001fc60028103fb000260018203fb0002019050fc
+```
+
+## Dispatch function
+
+### Pseudocode
+
+```
+function main()
+{
+    let i := shr(calldataload(4))
+    let sig := shr(calldataload(0), 224)
+    if (eq(sig, 0xc7665267)) // fac(uint256)
+    {
+      mstore(0, fac(i))
+      return(0, 32)
+    }
+    else if (eq(sig, 0xc6c2ea17)) // fib(uint256)
+    {
+      mstore(0, fib(i))
+      return(0, 32)
+    }
+    else
+    {
+      revert(0, 0)
+    }
+}
+```
+
+### Assembly
+
+```
+push1 4
+calldataload
+push1 0
+calldataload
+push1 0xe0
+shr
+push4 0xc7665267
+dup2
+eq
+rjumpi if1_true
+
+//if1_false:
```

Comment:
Why comment this out?

Reply:
I guess to not be mistaken for an opcode (no JUMPDEST needed for RJUMPs)